index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+        <header>
+            <h2 class="logo">My Logo</h2>
+            <img class="hamburger" src="images/hamburger-menu.png" alt="hamburger menu" width=60>
```

Comment:
Please ensure that this is a hamburger button, and should have a `nav` tag instead of an 'img' tag, you can use icons, you can get hamburger icons [HERE](https://fontawesome.com/v5.15/icons?d=gallery&p=1&q=Hamburger). And if you would like to use an image as your hamburger, then you should set it as a CSS background-image.